.github/pull_request_template.md

```diff
@@ -21,16 +21,19 @@ not fully covered by unit tests or manual testing can be complicated. -->
 <!-- Put an 'x' in all the boxes that apply -->
 - [ ] I have added unit tests to cover my changes.​
 - [ ] I have added integration tests to cover my changes.​
-- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
-- [ ] I have updated the [documentation](
-  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
+- [ ] I have added e2e tests for validation.
+- [ ] I have added the description of my changes into CHANGELOG in my target branch. (e.g., [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md) in develop)​
```

Comment:
This must be `I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md) in develop)​.`